modules/ossm-threescale-istio-adapter-troubleshooting.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v1x/threescale_adapter/threescale-adapter.adoc
+// * service_mesh/v2x/threescale_adapter/threescale-adapter.adoc
+
+[id="ossm-threescale-istio-adapter-troubleshooting_{context}"]
+= 3scale Istio adapter troubleshooting
+
+You might want to check whether the 3scale Istio adapter is working as expected. If this is not the case, you can troubleshoot from the list of scenarios.
+
+Perform the following steps to check that the adapter is working correctly:
+
+. Ensure the _3scale-adapter_ pod is running in the control plane namespace, that is <istio-system>.
+. Check that the _3scale-adapter_ pod has printed out information about itself booting up, such as its version.
```

Comment:
Where can you find this printout?  Is this a log?  Is there a console tab or view?  An output file?  Or is this under Status in a YAML file?

Reply:
@unleashed :point_up:

Reply:
Yes, this is a log - make explicit reference to printed out in the pod logs.